### PR TITLE
[WIP] Geoadmin with vector tile bg

### PR DESCRIPTION
**Pull request by oterral**
This PR is the base for the test viewer project

- [x] print is removed (is not yet on 
- [x] All bgs are replaced by 4 vector layers 
- [x] remove scale in footer

[Test](https://mf-geoadmin3.int.bgdi.ch/teo_wm_vt/index.html?topic=ech&lang=fr&bgLayer=sbm-osm&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,)

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/teo_wm_vt/index.html)</jenkins>

### CODE REVIEW

**Comment by davidoesch** [comment]
Can you remove the PRINT and DRAW/MEASURE accordeon?
(since for ALPHA this is not needed due to wrong results)

**Comment by oterral** [comment]
@davidoesch  print remove.

**Comment by davidoesch** [comment]
let me know when on https://mf-geoadmin3.int.bgdi.ch/teo_wm/index.html 
- [ ] print is removed (is not yet on 
- [ ] gey bg is replaced by a lightbasemap  or https://tileserver.dev.bgdi.ch/styles/swissrailway-basemap/

**Comment by oterral** [comment]
The branch is [teo_wm_vt](https://mf-geoadmin3.int.bgdi.ch/teo_wm_vt/index.html)

**Comment by oterral** [comment]
All done test link in description of the PR

**Comment by gjn** [comment]
This is so great! Thanks alot.

Remains the removal of the scale, then we can use it for demos.

**Comment by gjn** [comment]
https://mf-chsdi3.int.bgdi.ch/shorten/7a936c7d16

**Comment by p1d1d1** [comment]
Just amazing. .... _one small step for a man, one giant leap for mankind._

**Comment by davidoesch** [comment]
Don't be too enthusiastic. This is only the pre alpha for the stake holders to take a decision 👍 or no

**Comment by davidoesch** [comment]
Hi @oterral 

The https://mf-geoadmin3.int.bgdi.ch/teo_wm_vt/index.html?bgLayer=sbm-osm  won't load on edge explorer

**Comment by oterral** [comment]
It's working on my computer.

**Comment by davidoesch** [comment]
well 304 on mine using  RAS 

grafik

**Comment by oterral** [comment]
@gjn @davidoesch  fixed on ie 11 using a very ugly way :)

**Comment by gjn** [comment]
Master.

**Comment by gberaudo** [comment]
Regarding zoom experience of vector tiles, you have probably noticed the recent improvements by Andreas: https://github.com/openlayers/openlayers/pull/8227.

**Comment by oterral** [comment]
Yes I've seen but I haven't look  how to use the last version of ol yet

**Comment by fgravin** [comment]
Hi,

This is exactly what we've talked about at last foss4G and that would have been great to start with during the code sprint. Seems Andreas was a bit to excited of implementing it :)

Before: when zoom in, we wait to load the new vector tile then we generate the new image corresponding to new vector tile data, during all this process, the lower zoom image is still there and is ugly.

Now: while zooming and waiting the new vector tile to load and the new image to be rendered, we generate new images for higher zoom level using the previous loaded vector tiles data at a lower zoom level, so we generate lots of image to remove the ugly effect, but the images does not contain the zoom level data, untill the vector tile is loaded.

File: src/components/map/LayersService.js [hunk 1]
```diff
@@ -649,7 +729,107 @@ goog.require('ga_urlutils_service');
...
+                      });
+                      olLayer.setOpacity(config.opacity || 1);
+                    } else { // vector
+                      olSource = new ol.source.VectorTile({
+                        format: new ol.format.MVT(),
+                        minZoom: data.minzoom,
+                        maxZoom: data.maxzoom,
+                        urls: data.tiles
+                      });
+                    }
```
Comment (loicgasser) [inline src/components/map/LayersService.js]: I think we should use `ol.tile.TileJSON` and a dedicated service for loading/editing the glstyle.

see: https://openlayers.org/en/latest/apidoc/module-ol_source_TileJSON.html

**Pull request closed without merging**